### DISCUSSION TITLE
Custom theme

### DISCUSSION THREAD

**Discussion by WarriorHuangCheng** [element-plus/element-plus#12297] [Q&A]
Hi!
When will the ElementUI 3.0 version be able to use the online theme editor, which can modify and customize all global and component Design Tokens of Element, and conveniently preview the visual changes after style modification in real-time?
Like the ElementUI 2.x version at [https://element.eleme.io/#/en-US/theme](https://element.eleme.io/#/en-US/theme), you can use this link to customize some themes and preview them in real-time.

**Comment by reallyreallycai** [comment]
i am also looking forward to this

**Comment by ryuhangyeong** [comment]
It's likely to be released soon.

https://github.com/element-plus/element-plus/pull/9874
